typing_extensions/src/typing_extensions.py

```diff
@@ -1123,6 +1125,23 @@ class Point2D(TypedDict):
         """
 
 
+if hasattr(typing, "is_typeddict"):
+    is_typeddict = typing.is_typeddict
+else:
+    _TYPEDDICT_TYPES = (typing._TypedDictMeta, _TypedDictMeta) if hasattr(typing, "_TypedDictMeta") else (_TypedDictMeta,)
+
+    def is_typeddict(tp):
+        """Check if an annotation is a TypedDict class
+        For example::
+            class Film(TypedDict):
+                title: str
+                year: int
+            is_typeddict(Film)  # => True
+            is_typeddict(Union[list, str])  # => False
```

Comment:
```suggestion
        """Check if an annotation is a TypedDict class

        For example::
            class Film(TypedDict):
                title: str
                year: int

            is_typeddict(Film)  # => True
            is_typeddict(Union[list, str])  # => False
```
Restore spacing like in the CPython docstring